src/Azure.Deployments.Extensibility.Core.Tests.Unit/Azure - Backup.Deployments.Extensibility.Core.Tests.Unit.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
>Project [](http://example.com/codeflow?start=1&length=7)

Spaces in the file name intentional?

Reply:
Hmm I wonder why VS inserted the spaces. I created the project in VS without checking the file system. Let me fix this.

Reply:
Wait this is a back up file. Will remove it.